manage.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+"""To manage Blog app."""
```

Comment:
Not "To". You don't describe things this way.

Reply:
`Blog app manager.`

Reply:
I understood.